### DISCUSSION TITLE
Welcome to scam-site-s-source-code Discussions!

### DISCUSSION THREAD

**Discussion by tamagobuta** [tamagobuta/scam-site-s-source-code#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tamagobuta** [comment]
Hi

**Comment by hypnozzzzzz** [comment]
hello my Fortnite account was stolen I don't know what to do I warned everything I have proof and he changed my nickname so I had his irl who could help me I beg you

**Reply by tamagobuta** [reply]
I'm really sorry to hear about your situation with your Fortnite account. It's critical to act quickly in these cases. Start by reaching out to Epic Games support immediately. Provide them with all the evidence you have, including any details about the person who might have stolen your account, such as their in-real-life information. Epic Games support should be able to assist you in recovering your account or securing it. Remember to provide as much detailed information as possible to support your claim. Good luck!

**Comment by tamagobuta** [comment]
> hello my Fortnite account was stolen I don't know what to do I warned everything I have proof and he changed my nickname so I had his irl who could help me I beg you
